.circleci/cimodel/data/simple/android_definitions.py

```diff
@@ -0,0 +1,92 @@
+import cimodel.data.simple.util.branch_filters
+from cimodel.data.simple.util.docker_constants import DOCKER_IMAGE_NDK
+
+
+class AndroidJob:
```

Comment:
This is similar to https://github.com/pytorch/pytorch/blob/master/.circleci/cimodel/data/simple/nightly_android.py
Maybe we should consider merging these definitions in the future.